bitnami/argo-cd/templates/application-controller/sts-dpl.yaml

```diff
@@ -0,0 +1,224 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: {{ .Values.controller.enableStatefulSet | ternary "StatefulSet" "Deployment" }}
```

Comment:
What's the advantage of using StatefulSet instead of a Deployment? I don't see a volumeClaimTemplate or similar. What's the use case we're covering here?